### PR TITLE
update sample-controller README

### PR DESCRIPTION
**Pull request by imroc**
remind that CustomResourceSubresources is beta in v1.11 and enabled by default

### CODE REVIEW

**Comment by imroc** [comment]
corresponding official document: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources

File: staging/src/k8s.io/sample-controller/README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ Enable this feature using the `CustomResourceSubresources` feature gate on the [
 ```sh
 --feature-gates=CustomResourceSubresources=true
 ```
+It is beta in v1.11 and enabled by default.
```
Comment (neolit123) [inline staging/src/k8s.io/sample-controller/README.md]: better to first explain the state of the newer version?

---------------

Custom Resources support `/status` and `/scale` subresources as an
[beta feature](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#subresources) in v1.11 and is enabled by default.
The feature is alpha in in v1.10 and to enable it you need to set the `CustomResourceSubresources` feature gate on the [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver):

```sh
--feature-gates=CustomResourceSubresources=true
```

**Review by neolit123 — COMMENTED** [review]
added small note about the comment update. :+1:

File: staging/src/k8s.io/sample-controller/README.md (hunk 1 above)
Comment (imroc) [reply staging/src/k8s.io/sample-controller/README.md]: good idea. I'll fix it soon

**Comment by imroc** [comment]
@neolit123 I fixed it, PTAL.

File: staging/src/k8s.io/sample-controller/README.md [hunk 2]
```diff
@@ -105,9 +105,8 @@ $ kubectl create -f artifacts/examples/crd-validation.yaml
 
 ## Subresources
 
-Custom Resources support `/status` and `/scale` subresources as an
-[alpha feature](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#subresources) in v1.10.
-Enable this feature using the `CustomResourceSubresources` feature gate on the [kube-apiserver](https://kubernetes.io/docs/admin/kube-apiserver):
+Custom Resources support `/status` and `/scale` subresources as an [beta feature](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources) in v1.11 and is enabled by default.
```
Comment (neolit123) [inline staging/src/k8s.io/sample-controller/README.md]: oops my typo.
`as an` -> `as a`

Comment (imroc) [reply staging/src/k8s.io/sample-controller/README.md]: nice, fixed it, you're really careful, PTAL.

**Comment by nikhita** [comment]
@imroc thanks! :tada: 

/ok-to-test
/lgtm

/assign @sttts 
for approval

**Pull request merged**